Automattic/wp-calypso
# desktop: record e2e runs

### Description

Turns out e2e screen recording was inadvertently broken with the migration from the wp-desktop repository. This PR restores e2e capture, as well as enables recording on both Mac ~~and Linux platforms (previously Mac-only)~~. In addition to persisting app executables for inspection, screen captures allow us to inspect e2e test runs more conveniently in case of test failures.

_Update_: Seems that video capture in a Docker Linux instance is much less straightforward than Mac. Will revisit at a later date, but for now the implementation is Mac-only.

<p align="center">
<img width="500" alt="mac-e2e-artifacts" src="https://user-images.githubusercontent.com/8979548/91240116-13be8480-e70f-11ea-9906-7bdb1dc6e363.png">
</p>

### To Test

Verify the mpg and mp4 video files are present as Mac artifacts ~~and Linux artifacts, respectively~~.

Note: Use VLC - not Quicktime - to view these files. (Could not open the .mpg with Quicktime on my Mac, although YMMV.)
